composer.json

```diff
@@ -4,13 +4,14 @@
     "type": "composer-plugin",
     "license": "MIT",
     "require": {
-        "php": ">=5.6.0",
+        "php": "^7.0",
         "composer-plugin-api": "~1.0",
         "composer/composer": "^1.0",
         "doctrine/collections": "~1.2",
         "gitonomy/gitlib": "~1.0",
         "monolog/monolog": "~1.16",
-        "ocramius/proxy-manager": "~0.4|~1.0|~2.0",
+        "ocramius/proxy-manager": "~2.0.4",
+        "ocramius/package-versions": "~1.1",
```

Comment:
Why is package-versions added manually? We do not use it in this package.
FYI: We are also planning to drop the proxy-manager since it causes some installation issues.